<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>157</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Read Node Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="createButton">
      <property name="toolTip">
       <string>Create backdrops for all shots in the specified sequence directory</string>
      </property>
      <property name="text">
       <string>Create Shots</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="pathBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="rtdButton">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>background-color: darkRed; color: white;</string>
        </property>
        <property name="text">
         <string>Red to Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reloadButton">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Reload Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shotsButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>--Select Shots--</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="replaceButton">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="mainProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>shotsButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
